tests/queries/0_stateless/02305_schema_inference_with_globs.sh

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest
+# Tags: no-fasttest, no-parallel
```

Comment:
Guess we can remove `no-parallel` now after adding `CLICKHOUSE_TEST_UNIQUE_NAME`

Reply:
Great catch, I thought that I've removed it in the second patch, but seems forgot.